binder/postBuild

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# get rust
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+echo 'export PATH="$HOME/.cargo/bin:$PATH"' >> ~/.bashrc
+
+source ~/.bashrc
+
+# install prolog package
+cat << EOF > install.pl
+:- initialization main.
+
+install:-
+        pack_install(terminus_store_prolog).
+
+main:-
+        catch(install, E, (print_message(error, E), fail)),
+        halt.
+main:-
+        halt(1).
+EOF
+
+swipl install.pl
```

Comment:
Currently failing on my fumbling attempts to install the prolog package:
```

+ echo 'export PATH="$HOME/.cargo/bin:$PATH"'
+ cat
+ swipl install.pl
% Contacting server at http://www.swi-prolog.org/pack/query ...
ERROR: url `'https://www.swi-prolog.org/pack/query'' does not exist (status(500,Internal Server Error))
```

prolog and friends version installed via `apt`:
```

Selecting previously unselected package libxft2:amd64.
(Reading database ... 25575 files and directories currently installed.)
Preparing to unpack .../00-libxft2_2.3.2-1_amd64.deb ...
Unpacking libxft2:amd64 (2.3.2-1) ...
Selecting previously unselected package libxinerama1:amd64.
Preparing to unpack .../01-libxinerama1_2%3a1.1.3-1_amd64.deb ...
Unpacking libxinerama1:amd64 (2:1.1.3-1) ...
Selecting previously unselected package libarchive13:amd64.
Preparing to unpack .../02-libarchive13_3.2.2-3.1ubuntu0.6_amd64.deb ...
Unpacking libarchive13:amd64 (3.2.2-3.1ubuntu0.6) ...
Selecting previously unselected package libbsd-dev:amd64.
Preparing to unpack .../03-libbsd-dev_0.8.7-1ubuntu0.1_amd64.deb ...
Unpacking libbsd-dev:amd64 (0.8.7-1ubuntu0.1) ...
Selecting previously unselected package libedit-dev:amd64.
Preparing to unpack .../04-libedit-dev_3.1-20170329-1_amd64.deb ...
Unpacking libedit-dev:amd64 (3.1-20170329-1) ...
Selecting previously unselected package libjs-jquery.
Preparing to unpack .../05-libjs-jquery_3.2.1-1_all.deb ...
Unpacking libjs-jquery (3.2.1-1) ...
Selecting previously unselected package libossp-uuid16:amd64.
Preparing to unpack .../06-libossp-uuid16_1.6.2-1.5build4_amd64.deb ...
Unpacking libossp-uuid16:amd64 (1.6.2-1.5build4) ...
Selecting previously unselected package libxpm4:amd64.
Preparing to unpack .../07-libxpm4_1%3a3.5.12-1_amd64.deb ...
Unpacking libxpm4:amd64 (1:3.5.12-1) ...
Selecting previously unselected package swi-prolog-nox.
Preparing to unpack .../08-swi-prolog-nox_7.6.4+dfsg-1build1_amd64.deb ...
Unpacking swi-prolog-nox (7.6.4+dfsg-1build1) ...
Selecting previously unselected package swi-prolog-x.
Preparing to unpack .../09-swi-prolog-x_7.6.4+dfsg-1build1_amd64.deb ...
Unpacking swi-prolog-x (7.6.4+dfsg-1build1) ...
Selecting previously unselected package swi-prolog.
Preparing to unpack .../10-swi-prolog_7.6.4+dfsg-1build1_amd64.deb ...
Unpacking swi-prolog (7.6.4+dfsg-1build1) ...
Setting up libjs-jquery (3.2.1-1) ...
Setting up libxinerama1:amd64 (2:1.1.3-1) ...
Setting up libarchive13:amd64 (3.2.2-3.1ubuntu0.6) ...
Setting up libbsd-dev:amd64 (0.8.7-1ubuntu0.1) ...
Setting up libossp-uuid16:amd64 (1.6.2-1.5build4) ...
Setting up libxpm4:amd64 (1:3.5.12-1) ...
Setting up libedit-dev:amd64 (3.1-20170329-1) ...
Setting up libxft2:amd64 (2.3.2-1) ...
Setting up swi-prolog-nox (7.6.4+dfsg-1build1) ...
update-alternatives: using /usr/bin/swipl to provide /usr/bin/prolog (prolog) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/prolog.1.gz because associated file /usr/share/man/man1/swipl.1.gz (of link group prolog) doesn't exist
Setting up swi-prolog-x (7.6.4+dfsg-1build1) ...
Setting up swi-prolog (7.6.4+dfsg-1build1) ...
Processing triggers for libc-bin (2.27-3ubuntu1.2) ...
```

```

Reply:
rebuilding `swi-prolog` from source "fixed" this...